src/desktop/apps/consign/components/describe_work_container/index.js

```diff
@@ -5,9 +5,11 @@ import { formattedLocation } from "../../helpers"
 import { makeDescribeWorkDesktop } from "../describe_work_desktop"
 import { makeDescribeWorkMobile } from "../describe_work_mobile"
 import { isEmpty, pick } from "underscore"
+import { data as sd } from "sharify"
 
 function DescribeWorkContainer(props) {
-  const { isMobile, phone, submission } = props
```

Comment:
Is there infrastructure to include the `sd` values in the props passed into this component? The obvious-ness of this value never being set makes me think it was expected to be passed in.

Reply:
In general this is true, as we can check the `res.locals.IS_MOBILE` flag of the express request and pass the value into a SSR-rendered app, but it looks like the React aspect of this (the consignments flow) is [client-side only](https://github.com/damassi/force/blob/0721f77ebfd220bebb7accaf411edc242760617f/src/desktop/apps/consign/routes.js#L75), attaching to [this template](https://github.com/damassi/force/blob/0721f77ebfd220bebb7accaf411edc242760617f/src/desktop/apps/consign/templates/submission_flow.jade#L12). In this case, using `sd.IS_MOBILE` will work aok.